### PR TITLE
kernel: remove build-kernel_test.sh

### PR DESCRIPTION
**Pull request by chavafg**
`build-kernel_test.sh` builds the kernel when there are changes
of it in a PR and then runs the whole CI tests. Now we are running
all CI tests on all changes[1] (not only when there is a kernel change).
This is making the CI to run all tests twice when there is a change
in the kernel, so we need to remove the script.

[1] https://github.com/kata-containers/packaging/pull/348

Fixes: #380.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

File: kernel/kata_config_version [hunk 1]
```diff
@@ -1 +1 @@
-26
+27
```
Comment (grahamwhaley) [inline kernel/kata_config_version]: Do you need this bump - you didn't change the kernel version or config file for instance, so the kernel we are building is the same...?

Comment (chavafg) [reply kernel/kata_config_version]: Yeah, but the CI script asks for a bump when the `kernel/` directory gets modified. Maybe we should improve this on a following PR.

Comment (grahamwhaley) [reply kernel/kata_config_version]: ah, OK, fair enough. thx

**Pull request merged**